spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapConfigProperties.java

```diff
@@ -170,34 +184,16 @@ public boolean isEmpty() {
 		public NormalizedSource normalize(String defaultName, String defaultNamespace) {
 			String normalizedName = StringUtils.hasLength(this.name) ? this.name : defaultName;
 			String normalizedNamespace = StringUtils.hasLength(this.namespace) ? this.namespace : defaultNamespace;
-			return new NormalizedSource(normalizedName, normalizedNamespace, "");
+			return new NormalizedSource(normalizedName, normalizedNamespace, "", true);
 		}
 
-		public NormalizedSource normalize(String defaultName, String defaultNamespace, boolean defaultUseNameAsPrefix) {
+		public NormalizedSource normalize(String defaultName, String defaultNamespace, boolean defaultUseNameAsPrefix,
+				boolean defaultUseProfileNameAsSuffix) {
 			String normalizedName = StringUtils.hasLength(this.name) ? this.name : defaultName;
 			String normalizedNamespace = StringUtils.hasLength(this.namespace) ? this.namespace : defaultNamespace;
-
-			// if explicitPrefix is set, it takes priority over useNameAsPrefix
```

Comment:
not deleted, but verbatim moved. There is a diff PR that does only this, once that is merged and I rebased, this change will not be visible in this PR anymore